### ISSUE TITLE
Problem with multiple controlnet

### ISSUE THREAD

**Issue by JerryOver** [Mikubill/sd-webui-controlnet#691]
I have used multiple controlnets in the past. I usually run camenduru's colab, but for about a week, I have been having trouble running multiple controlnets. Today was the first time I tried using nolanaatama's colab, and I had issues with that one too. What I do is I go to settings and increase the amount of controlnets. I then TRY to reload the WebUI, but it never runs. LIKE USUAL. I STOP the colab and press PLAY again. When the new link is ready, I load up the new WebU. There are multiple controlnets, but they seem to malfunction. For the most part, I can't get more than one controlnet running simultaneously. For example, I can't get a canny and a depth to run at the same time, AND YES THEY ARE BOTH ENABLED. I have in the past. In the past I have ran 5 simultaneously: canny,depth,depthleres,hed, and mlsd. I guess it has been about a week or so that they have stopped performing well. I have really been wanting to try out clip vision but haven't had the chance yet.

**Comment by ljleb** [comment]
Do you see any output in the colab console? It would help with debugging.

**Comment by JerryOver** [comment]
I will try to run it again today to let you know. Usually the first controlnet will run fine with either canny or depth but not always. Sometimes it will say something like no such model, but if I keep trying, it will eventually start working. I have recently had trouble just running two. It doesn't seem to be working like it use to, but I will run it today to let you know specifically what's up.

**Comment by JerryOver** [comment]
I will try to run it again today to let you know. Usually the first controlnet will run fine with either canny or depth but not always. Sometimes the command line window will say something like no such model, but if I keep trying, it will eventually start working with two outputs generated in the console. For the most part, I have recently had trouble running two controlnets. In the past I have run five simultaneously. It doesn't seem to be working like it use to, but I will run it today to let you know specifically what the command line and console are doing.

**Comment by JerryOver** [comment]
So I just ran Camenduru's [nightly] stable_diffusion_1_5_webui_colab. Once the console was loaded up, I changed the settings to create 6 controlnets and reloaded the WebUI. All 6 were there after reload, but I only used the first two controlnets within the TextToImage part of WebUI. the two controlnets were of course both enabled and had images  . The first was canny and the second was clip vision with style adapter. NOTHING HAPPENED after the first time I clicked GENERATE. There was an image, but it had nothing to do with what was going in controlnet. There was also no prompt because based on the Youtube tutorial I watched I was just trying to influence image in the first controlnet mini-window with the style of the image in the second controlnet mini-window. There was suppose to be a canny image generated too, but there wasn't one. There was just one image that looked nothing like the image in the first controlnet mini-window. I then went back to both mini-windows for both controlnets. In the first I changed it from canny to depth just to see the loading bar appear in the mini-window; I did the same for the second controlnet mini-window but kept it on clipvision and the style adapter. It ran but the console showed a canny image instead of a depth image. I got an output image too, but I'm not so sure if it was what it was suppose to be based on the Youtube tutorial. I ran it again keeping everything the same, the console had an output image and a depth image this time. The output didn't look like what I expected it to be based on a Youtube tutorial I watched, so I ran it again keeping the two controlnets on the same settings: Controlnet1 - depth & Controlnet2 - clipvision with style adapter. NOTHING GENERATED AT ALL this time. I ran it again and got the exact same results. The images below show what the command line window said. The first one is what it said at the very beginning. The second image is what it said at the end. It was rather long.
 
![4](https://user-images.githubusercontent.com/120525455/229323494-fb3796aa-6959-409a-be0f-2514bba17ffd.png)

![5](https://user-images.githubusercontent.com/120525455/229323496-729bef1a-a5e7-4015-acb2-60b97c234fc5.png)

I think I got this error before. The other day I even tried to run JUST depth and canny together. I had a prompt that time. Only the first controlnet mini-window worked. The second seemed to malfunction. I have ran 6 controlnets simultaneously before, but it doesn't seem to be working for me like it once did.
